gh_inspector.gemspec

```diff
@@ -23,4 +23,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rake', '~> 10.0'
   spec.add_development_dependency 'rspec', '~> 3.0'
   spec.add_development_dependency 'rubocop', '~> 0', '> 0'
+  spec.add_development_dependency 'yard', '~> 0', '> 0'
```

Comment:
Not sure what `'~> 0', '> 0'` means but thought it allows any version, obviously copied from above.